docs/backends/oracle.rst

```diff
@@ -514,6 +515,16 @@ plan to manage zones with the **pdnsutil** tool. Default:
 
     UPDATE ZoneDNSKeys SET active = :active WHERE id = :keyid
 
+oracle-set-zone-key-hidden-query
+''''''''''''''''''''''''''''''''
+
+Hide or unhide a DNSSEC signing key. You can skip this if you do not
+plan to manage zones with the **pdnsutil** tool. Default:
```

Comment:
as a general not, pdnsutil spawns a backend (via the UeberBackend) and uses those semantics to hide/unhide a key, so the query should be a working one.

Reply:
This was copy pastad from the "oracle-set-zone-key-state-query" bit, should that be changed as well?

Reply:
Since the oracle backend is gone I'll resolve this discussion :P